example_sns_init.yaml

```diff
@@ -83,19 +83,22 @@ description: >
 # cropped to a circle.
 logo: logo.png
 
-# URL to the dapp controlled by the SNS project.
-# Must be a string from 10 to 512 bytes.
-url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
+# URL that represents the SNS, usually a dapp controlled by the SNS project.
+# Must be a string from 10 to 512 bytes, and the domain must be HTTPS.
+url: https://mydapp.com
 
 # Metadata for the NNS proposal required to create the SNS. This data will be
 # shown only in the NNS proposal.
 NnsProposal:
     # The title of the NNS proposal. Must be a string of 4 to 256 bytes.
     title: "NNS Proposal to create an SNS named 'Rock Out'"
 
-    # The HTTPS address of additional content required to evaluate the NNS
+    # The HTTPS address of a forum post with additional context required to evaluate the NNS
```

Comment:
does not have to be forum post, could be white paper, I think. Cc @LaraAS

Reply:
I now see that we have `allowed_domains` set to "forum.dfinity.org". So nevermind.